docs/syz_verifier.md

```diff
@@ -0,0 +1,81 @@
+# syz-verifier
+
+Logical bugs are a type of  bugs that make a program operate incorrectly, possibly
```

Comment:
I think now that this is becoming a public document, we should be more precise.

My suggestion:
```
Testing and fuzzing aims to verify a subset of the behaviour of running programs: any logical error can cause bugs that make the program behave incorrectly, possibly producing unintended output. Many bugs, however, do not cause the program to crash nor produce assertion failures. Here we need to distinguish between undefined and defined behaviour. 

Undefined behaviour can easily be detected by either static (e.g. compiler warnings) or dynamic analysis (e.g. sanitizers). However, logical errors that result in defined program behaviour, yet are still causing bugs that make the program behave incorrectly according to some documented or otherwise expected intent, are much harder to detect. One way to detect such bugs is setting up test cases that compare outputs against a set of expected results.
```

Reply:
Thank you!